sdk/device.go

```diff
@@ -709,6 +715,10 @@ type DeviceInstance struct {
 	// DeviceKind.
 	DisableOutputInheritance bool `yaml:"disableOutputInheritance,omitempty" addedIn:"1.0"`
 
+	// SortOrdinal is a one based sort ordinal for a device in a scan. Zero for
+	// don't care.
+	SortOrdinal int32
```

Comment:
we should probably add an annotation to this field to be consistent
```
SortOrdinal int32 `yaml:"sortOrdinal,omitempty" addedIn:"1.0"`
```

Reply:
I'll add that